examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant/README.md

```diff
@@ -40,17 +40,17 @@ python benchmark.py \
 
 
 # Validated Models
-| Models\Acc       |  FP32  |    int8     | SQ           |
```

Comment:
Acc -> Accuracy

Add a reference to SmoothQuant paper at the bottom https://arxiv.org/abs/2211.10438

Reply:
ref paper already in the bottom of this page